packages/melos/lib/src/workspace_configs.dart

```diff
@@ -418,10 +419,16 @@ class MelosWorkspaceConfig {
       map: yaml,
     );
 
+    final sdkPath = assertKeyIsA<String?>(
+      key: 'sdk_path',
```

Comment:
`sdk_path` -> `sdkPath`

Reply:
Fixed in ebe5de74daa428593bc131295a89ca342a7ebcf1